prisma/schema.prisma

```diff
@@ -51,15 +51,34 @@ model User {
   telegram            String?     @db.VarChar
   github              String?     @db.VarChar
   hashed_ip_address   String?     @db.VarChar
-  events              Event[]
-  node_uptime         NodeUptime?
-  points              UserPoints?
+  redemption          Redemption?
+
+  events      Event[]
+  node_uptime NodeUptime?
+  points      UserPoints?
 
   @@index([email], name: "index_users_on_email")
   @@index([graffiti], name: "index_users_on_graffiti")
   @@map("users")
 }
 
+model Redemption {
```

Comment:
we could just call this kyc

Reply:
No, they are not the same thing. See spec.